grafana_wtf/core.py

```diff
@@ -492,6 +492,36 @@ def explore_dashboards(self, with_data_details: bool = False):
 
         return results
 
+    def plugins_list(self):
+        return self.grafana.plugin.get_installed_plugins()
+
+    def plugins_status(self):
+        status = []
+        plugins = self.grafana.plugin.get_installed_plugins()
+        for plugin in plugins:
+            plugin = munchify(plugin)
+            item = Munch(
```

Comment:
I think we can filter out the panel type as they don't have server to respond from.